### DISCUSSION TITLE
development of shopping cart and checkout block

### DISCUSSION THREAD

**Discussion by KulacsMarket** [woocommerce/woocommerce#49258] [Checkout Flow]
"Feedback?
We are currently working on developing the cart and checkout blocks to provide merchants with the tools and customization options they need.

Tell me what you think." - When are these blocks expected to be customizable? What other solution is there?

**Comment by jocala99** [comment]
+1. I would like to modify the display of the Cart block to simply change the heading "Products" to the word "Items." I can find no way to do this.

**Reply by nielslange** [reply]
> I would like to modify the display of the Cart block to simply change the heading "Products" to the word "Items." I can find no way to do this.

For visibility-reasons, this question had been answered in https://github.com/woocommerce/woocommerce/discussions/47921#discussioncomment-10052337.

**Comment by nielslange** [comment]
> We are currently working on developing the cart and checkout blocks to provide merchants with the tools and customization options they need.

> When are these blocks expected to be customizable? What other solution is there?

@KulacsMarket Please provide more details on the issue you are facing. What exactly are you trying to accomplish, and what would you need to accomplish that?
